src/components/Profile/components/EmergencyInfoList/index.js

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+import './index.css';
+import ProfileInfoListItem from '../ProfileInfoListItem';
+import user from 'services/user';
+
+import { Typography, Grid, Card, CardHeader, CardContent, List } from '@material-ui/core';
+import { useEffect } from 'react';
+
+const formatPhone = (phone) => {
+  let tele = String(phone);
+  if (tele.length === 10) {
+    return '(' + tele.slice(0, 3) + ') ' + tele.slice(3, 6) + '-' + tele.slice(6);
+  }
+  if (tele !== 'undefined') {
```

Comment:
```suggestion
  else if (tele !== 'undefined') {
```

Reply:
Yeah, this method is somewhat confusing. Because I copied the whole method from somewhere else in UI, I suppose I should also update the other one.